ghrn/ghrn.go

```diff
@@ -13,96 +13,118 @@ import (
 
 // Config describes configuration for BuildReleaseNotes.
 type Config struct {
-  // Org is the name of the GitHub organization. Required.
-  Org string
-  // Repo is the name of the GitHub repository. Required.
-  Repo string
-
-  // GitHubToken is a GitHub API access token.
-  GitHubToken string
-
-  // StopAt is the number of the Pull Request to stop at.
-  // Useful for building the notes of PRs since the last release, for example.
-  StopAt int
-  // IncludeCommits will include commmits messages for each PR.
-  IncludeCommits bool
+	// Org is the name of the GitHub organization. Required.
+	Org string
+	// Repo is the name of the GitHub repository. Required.
+	Repo string
+
+	// GitHubToken is a GitHub API access token.
+	GitHubToken string
+
+	// StopAt is the number of the Pull Request to stop at.
+	// Useful for building the notes of PRs since the last release, for example.
+	StopAt int
+	// IncludeCommits will include commmits messages for each PR.
+	IncludeCommits bool
+	// StopAtLatestRelease will stop at latest release commit.
```

Comment:
Let's call this "SinceLatestRelease" to match the flag. Docs would be "SinceLatestRelease will only include PRs and commits merged since the latest release tag."